scripts/cmake/CompilerSetup.cmake

```diff
@@ -135,7 +135,8 @@ if(WIN32)
             /wd4267 /wd4996 /bigobj")
         set(CMAKE_CXX_FLAGS_RELWITHDEBINFO  "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} \
             /ZI /Od /Ob0")
-
+        set(CMAKE_EXE_LINKER_FLAGS_DEBUG "${CMAKE_EXE_LINKER_FLAGS_DEBUG}  /ignore:4099")
```

Comment:
I know, that other magic numbers used in disabling of warnings etc. are not explained, but it would be a good starting point to add a short comment about 4099